pkg/services/k8s/apiserver/admission/add_default_fields.go

```diff
@@ -39,6 +40,10 @@ const (
 func (addDefaultFields) Admit(ctx context.Context, a admission.Attributes, o admission.ObjectInterfaces) (err error) {
 	obj := a.GetObject()
 
+	if true {
+		return nil
```

Comment:
@charandas -- this was failing in some cases, so lets comment it out until we have a real use for it